README.md

```diff
@@ -1,63 +1,124 @@
-> "Well, in our country," said Alice, still panting a little, "you'd generally 
-> get to somewhere else—if you run very fast for a long time, as we've been 
+> "Well, in our country," said Alice, still panting a little, "you'd generally
+> get to somewhere else—if you run very fast for a long time, as we've been
 > doing."
 >
-> "A slow sort of country!" said the Queen. "Now, here, you see, it takes all 
-> the running you can do, to keep in the same place. If you want to get 
+> "A slow sort of country!" said the Queen. "Now, here, you see, it takes all
+> the running you can do, to keep in the same place. If you want to get
 > somewhere else, you must run at least twice as fast as that!"
 >
 > [Carroll, Lewis: Through the Looking-Glass, Chapter 2](
     https://www.gutenberg.org/files/12/12-h/12-h.htm)
 
-## About
+<br>
+
+<h1>About</h1>
+
 The Red Queen benchmark framework was created to facilitate the benchmarking
 algorithms used in quantum compilation.
 
-The framework is tightly integrated into `pytest`.  Therefore, to use it 
-effectively, you should know the basics of `pytest` first. Take a look at the 
+The framework is tightly integrated into `pytest`.  Therefore, to use it
+effectively, you should know the basics of `pytest` first. Take a look at the
 [introductory material](https://docs.pytest.org/en/latest/getting-started.html).
 
-## Usage
+
+<br>
+
+<h1>Usage</h1>
```

Comment:
Does using explicit html markup for headers and breaks improve the formatting here? I think we should stick to the markdown formatting if not.

Reply:
I was looking into it, and it doesn't really add much. The Web Dev in me wanted it to make sense, but it doesn't. I'll adjust it accordingly.